lib/new_relic/agent/configuration/default_source.rb

```diff
@@ -1595,6 +1595,15 @@ def self.enforce_fallback(allowed_values: nil, fallback: nil)
           :allowed_from_server => false,
           :description => 'Controls auto-instrumentation of Sinatra at start up. May be one of: `auto`, `prepend`, `chain`, `disabled`.'
         },
+        :'instrumentation.stripe' => {
+          :default => 'enabled',
+          :documentation_default => 'enabled',
+          :public => true,
+          :type => String,
+          :dynamic_name => true,
```

Comment:
Your new `lib/new_relic/agent/instrumentation/stripe.rb` file uses the `'instrumentation.stripe'` string statically, so there's no need for the `dynamic_name` key/value pair here. Rather than switch the value to `false`, you can just remove both the key and value.

Reply:
Alternatively, you might be able to keep `dynamic_name: true` here, and remove the potentially redundant `depends_on` check in the new instrumentation class.

Reply:
I see how there was no need for the dynamic naming, so just removed it ◡̈ https://github.com/newrelic/newrelic-ruby-agent/pull/2180/commits/135282a44fd26491287f8389645522497145eb77